src/main/java/seedu/address/model/Model.java

```diff
@@ -133,4 +136,14 @@
      * @throws NullPointerException if {@code predicate} is null.
      */
     void updateFilteredSlotList(Predicate<Slot> predicate);
+
+    /**
+     * Returns an unmodifiable view of the filtered list of display items.
+     */
+    ObservableList<DisplayItem> getFilteredDisplayList();
+
+    /**
+     * Changes the list to be displayed.
+     */
+    void changeDisplaySystem(DisplaySystemType newDisplayType) throws IllegalValueException;
```

Comment:
We discussed this but kept only one Model because only ModelManager implements Model in the code.
But in tests model stubs implement model so every time you add a method to model you not only have to change ModelManager but also all the stubs in the tests.

I think it's better to only have to pass around one model so we just have to bear with this . . .

Reply:
uhh not very sure what you mean so what is the alternative?